README.md

```diff
@@ -29,6 +29,8 @@ To build RustPython locally, do the following:
     $ git clone https://github.com/RustPython/RustPython
     $ cd RustPython
       # --release is needed (at least on windows) to prevent stack overflow
+
+    $ curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
Installing `rustup` is only applicable for those who have not already installed it. Perhaps this should be moved to before the `git clone` step with an applicable comment.

Reply:
I also want to link https://rustup.rs website instead of putting the script here. The script is various depending on running platforms